src/pages/Auth/Logout.vue

```diff
@@ -31,11 +31,10 @@ export default {
     }
   },
   methods: {
-    ...mapActions('auth', ['login']),
     redirectToLogin() {
       this.redirecting = true
       setTimeout(() => {
-        this.login()
+        window.location.href = '/'
```

Comment:
Since login now happens automatically on application start, we can redirect back to the root directory to start the process.